library_generation/generate_library.sh

```diff
@@ -107,9 +107,20 @@ mkdir -p "${output_folder}/${destination_path}"
 # get a fixed order.
 folder_name=$(extract_folder_name "${destination_path}")
 pushd "${output_folder}"
-proto_files=$(find "${proto_path}" -type f  -name "*.proto" | sort)
+find_depth=""
+case "${proto_path}" in
```

Comment:
How many services we need to search for subdirectories? If there is only a few, maybe we should make searching for current folder the default?Because in some cases it could be wrong to generate for subdirectories.

For example, [fleetengine/v1](https://github.com/googleapis/googleapis/tree/master/google/maps/fleetengine/v1) and [fleetengine/delivery/v1](https://github.com/googleapis/googleapis/tree/master/google/maps/fleetengine/delivery/v1) are two separate apis.

Reply:
> How many services we need to search for subdirectories? 

Less than 5 libraries, aiplatform, for example.

Though in this PR, all special cases are common protos and I'm not sure whether subdirectory protos are included in service clients.

> For example, [fleetengine/v1](https://github.com/googleapis/googleapis/tree/master/google/maps/fleetengine/v1) and [fleetengine/delivery/v1](https://github.com/googleapis/googleapis/tree/master/google/maps/fleetengine/delivery/v1) are two separate apis.

I didn't test this case because they are self-service client.